src/cf-forms/src/molecules/form-fields.less

```diff
@@ -101,4 +101,73 @@
             }
         }
     }
+
+    &__select {
+        .a-select {
+            position: relative;
+            margin-top: 0.5em;
+            border: 1px solid @select-border;
+
+            select {
+                height: unit( 30px / @base-font-size-px, em );
+                width: 100%;
+                padding: unit( 4px / @base-font-size-px, em )
+                         unit( 6px / @base-font-size-px, em );
+                border: 0;
+                outline: 2px solid transparent;
+                appearance: none;
+                background-color: @input-bg;
+                border-radius: 0;
+                color: @text;
+
+                &:hover,
+                &:active,
+                &:focus {
+                    outline-color: @input-border__focused;
+                    outline-offset: 0;
+                }
+            }
+
+            select[disabled] {
+                color: @input-text__disabled;
+                background-color: @input-bg__disabled;
+                cursor: not-allowed;
+
+                &:hover {
+                    outline: 2px solid transparent;
```

Comment:
That's not an option with outline and you want there to be that offset so that the select doesn't "jump" on hover.